packages/camera/camera_android_camerax/example/lib/main.dart

```diff
@@ -2,43 +2,1003 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:async';
```

Comment:
This is an exact copy from [`camera_android`](https://github.com/flutter/plugins/blob/main/packages/camera/camera_android/example/lib/main.dart), **except** some functionality was deleted for currently unsupported operations. My thought is that we can just fill them back in as we implement things. I left TODOs in those spots.